spec/classes/puppet_config_spec.rb

```diff
@@ -1,18 +1,8 @@
 require 'spec_helper'
 
 describe 'puppet::config' do
-  on_supported_os.each do |os, os_facts|
-    next if only_test_os() and not only_test_os.include?(os)
-    next if exclude_test_os() and exclude_test_os.include?(os)
+  on_os_under_test.each do |os, facts|
     context "on #{os}" do
-      let (:default_facts) do
-        os_facts.merge({
-          :concat_basedir => '/foo/bar',
-          :domain         => 'example.org',
```

Comment:
I think this might be required to make the tests pass, it looks like srv_domain is set in puppet.conf from $::domain.